extensions/applicationinsights-dependencies-js/src/ajax.ts

```diff
@@ -322,7 +322,9 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
     /// <returns>True if Ajax monitoring is supported on this page, otherwise false</returns>
     private supportsAjaxMonitoring(): boolean {
         let result = true;
-        if (CoreUtils.isNullOrUndefined(XMLHttpRequest) ||
+
+        if (typeof XMLHttpRequest !== 'function' ||
```

Comment:
The ajax tests are failing because XMLHttpRequest is not a function for some versions of IE
```suggestion
        if (typeof XMLHttpRequest === 'undefined' ||
```